xapian-core/tests/api_db.cc

```diff
@@ -1687,14 +1687,54 @@ DEFINE_TESTCASE(sortrel1, backend) {
     return true;
 }
 
-// Test network stats and local stats give the same results.
-DEFINE_TESTCASE(netstats1, remote) {
-    XFAIL_FOR_BACKEND("multi_glass_remoteprog_glass",
-		      "Multi remote databases are currently buggy");
+static void
+make_netstats1_db(Xapian::WritableDatabase& db, const string&)
+{
+    struct { Xapian::docid did; const string text; } content[] = {
```

Comment:
We don't need to be able to modify this, so if you make this `static const` and use `const char* text` then the compiler should be able to put this in the read-only data section at compile time.

If you use `string` here, that prevents that happening (at least with current compilers - I guess theoretically they could spot that compile-time construction is possible).